discover/deployment/articles-dxp/03-maintaining-liferay/01-patching-liferay/01-patching-tool-configuration/01-configuration-basic.markdown

```diff
@@ -0,0 +1,39 @@
+# Patching Tool
```

Comment:
When I go through this, I'll probably re-work some of these headings according to our [guidelines](https://github.com/liferay/liferay-docs/blob/master/guidelines/writers-guidelines.markdown).